extensions/amp-story/1.0/media-tasks.js

```diff
@@ -109,200 +108,132 @@ function copyAttributes(fromEl, toEl) {
 }
 
 /**
- * Base class for tasks executed in order on HTMLMediaElements.
+ * A task executed in order on HTMLMediaElements.
+ * @interface
```

Comment:
Making this an interface so that it doesn't have to be extended and can be removed from bundle.